src/jbi/bugzilla_objects.py

```diff
@@ -0,0 +1,244 @@
+    """Bugzilla Comment Object"""
+
+    body: Optional[str]
+    id: Optional[int]
+    number: Optional[int]
+    is_private: Optional[bool]
+    creation_time: Optional[datetime.datetime]
+
+    def is_comment_description(self) -> bool:
+        """Used to determine if `self` is a description or comment."""
+        return self.number == 0
+
+    def is_comment_generic(self) -> bool:
+        """All comments after comment-0 are generic"""
+        is_description = self.is_comment_description()
+        return not is_description
+
+    def is_private_comment(self) -> bool:
+        """Helper function to determine if this comment private--not accessible or open"""
+        return bool(self.is_private)
+
+
+class BugzillaBug(BaseModel):
+    """Bugzilla Bug Object"""
+
+    id: int
+    is_private: Optional[bool]
+    type: Optional[str]
+    product: Optional[str]
+    component: Optional[str]
+    whiteboard: Optional[str]
+    keywords: Optional[List]
+    flags: Optional[List]
+    status: Optional[str]
+    resolution: Optional[str]
+    see_also: Optional[List]
+    summary: Optional[str]
+    severity: Optional[str]
+    priority: Optional[str]
+    creator: Optional[str]
+    assigned_to: Optional[str]
+    comment: Optional[BugzillaWebhookComment]
+
+    def get_whiteboard_count(self):
+        """Get count of whiteboard labels"""
+        if self.whiteboard is not None:
+            return self.whiteboard.count("[")
+        return 0
+
+    def get_whiteboard_as_list(self):
+        """Convert string whiteboard into list, splitting on ']' and removing '['."""
+        if self.whiteboard is not None:
+            split_list = self.whiteboard.replace("[", "").split("]")
+            return [x.strip() for x in split_list if x not in ["", " "]]
+        return []
+
+    def get_whiteboard_with_brackets_as_list(self):
+        """Convert string whiteboard into list, splitting on ']' and removing '['; then re-adding."""
+        wb_list = self.get_whiteboard_as_list()
+        if wb_list is not None and len(wb_list) > 0:
+            return [f"[{element}]" for element in wb_list]
+        return []
+
+    def get_jbi_labels(self):
+        """
+        whiteboard labels are added as a convenience for users to search in jira;
+        bugzilla is a required label for SYNC project
+        """
+        return (
+            ["bugzilla"]
+            + self.get_whiteboard_as_list()
+            + self.get_whiteboard_with_brackets_as_list()
+        )
+
+    def get_potential_whiteboard_config_list(self):
+        """Get all possible whiteboard_tag configuration values"""
+        converted_list: List = []
+        for whiteboard in self.get_whiteboard_as_list():
+            converted_tag = self.convert_wb_to_tag(whiteboard=whiteboard)
+            if converted_tag not in [None, "", " "]:
+                converted_list.append(converted_tag)
+
+        return converted_list
+
+    @staticmethod
+    def convert_wb_to_tag(whiteboard):
```

Comment:
Why is this static? Do we call this from elsewhere?

Also, other methods have `whiteboard` in their name, why not this one?